integration_tests/src/test_util.rs

```diff
@@ -120,132 +120,6 @@ impl log::Log for FileLogger {
 
 // ================== Test Helpers ================== //
 
-// FIXME: We actually probably want to use this. Figure out later.
-/// The suite of tests to run against a fully-instantated and storage-backed directory.
-/// This will publish 3 epochs of ```num_users``` records and
-/// perform 10 random lookup proofs + 2 random history proofs + and audit proof from epochs 1u64 -> 2u64
-#[allow(unused)]
-pub(crate) async fn directory_test_suite<
```

Comment:
This test is already present in `akd_test_tools` and is used from there.

https://github.com/novifinancial/akd/blob/9e2ef7e2518da9b16159351e911c8438b3d0ce00/integration_tests/src/memory_tests.rs#L23

https://github.com/novifinancial/akd/blob/9e2ef7e2518da9b16159351e911c8438b3d0ce00/integration_tests/src/mysql_tests.rs#L51